pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -680,6 +680,17 @@ type PoolSpec struct {
 	// +optional
 	// +nullable
 	Quotas QuotaSpec `json:"quotas,omitempty"`
+
+	// The pg settings
+	// +optional
+	// +nullable
+	PGs int `json:"pgs,omitempty"`
```

Comment:
We should consider using `PlacementGroupCount`. K8s API standards suggest that naming conventions should be explicit. This will help users understand the field when reading manifests if they aren't familiar with the PG acronym.

```suggestion
	PlacementGroupCount int `json:"placementGroupCount,omitempty"`
```

Reply:
@travisn can we use the `Parameters` to set `pg_autoscale_mode`?
https://docs.ceph.com/en/latest/rados/operations/placement-groups/

What about for `pg_num`?

Essentially, can we accomplish this without code changes and maybe just documenting this particular usage pattern? I would definitely consider this an advanced configuration, and if we can use `Parameters` for this, that seems entirely appropriate.

Reply:
@travisn @BlaineEXE is it suitable to set it to `Parameters` or add new schema

Reply:
There was a separate related discussion [here](https://github.com/rook/rook/pull/11967#discussion_r1153714677). We didn't see a way to set the autoscaleMode without the parameter at pool creation time. So the PGs are just an example in the yaml, and no update to the schema.